starter-code/css/main.css

```diff
@@ -6,3 +6,148 @@ Green: #00B172
 White: #FFF
 
 */
+
+body {
+    margin: 0;
+    padding: 0;
+}
+
+.nav-bar {
+    display: flex;
+    justify-content: space-around;
+    width: 100%;
+    align-items: flex-end;
+    
+    
+}
+  
+.nav-bar img {
+    display: flex;
+    justify-content: space-between;
+    float: left;
+    height: 100%;
+    width: 25%;
+    margin-bottom: 30px;
+    margin-top: 20px;
+}
+ul {
+    list-style-type: none;
+}
+  
+.navigation  {
+    display: flex;
+    justify-content: space-between;
+    flex-direction: row;
+}
+.navigation li {
+    padding: 30px;
+    color: black;
+    font-size: 2em;
+}
+
+
+.text-block {
+    position: absolute;
+    top: 70%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    color: white;
+    font-size: 30px;
+    text-align: center;
+  }
+
+.main-image img {
+    /* background-image: url(../images/landing.jpg);
+    background-repeat: no-repeat; */
+    height: 900px;
+    width: 100%;
+    
+}
+.main-image {
+   
+    background-image: url(../images/landing.jpg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    width: 100%;
+    height: 900px;
+}
+
+
+h2 {
+    font-size: 30px;
+    text-align: center;
+    margin: 0;
+    padding-top: 30px;
+}
+hr {
+    border-top: 4px solid #00B172;
+    width: 15%;
+    margin: 0, 30%;
+    text-align: center;
+}
+
+.capacity .item{
+    display:flex;
+    justify-content: space-around;
+    flex-wrap: wrap;
+    flex-direction: column;
+    font-size: 25px;
+    align-items: center;
+}
+.capacity {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    
+}
+
+.capacity img {
+    width: 150px;
+    height: 100px;
+    justify-content: space-around;
+    display: flex;
+    
+    
+}
+.capacity h2 {
+    font-size: 30px;
+    color: #00B172;
+}
+
+.green-box {
+    margin: 40px;
+    color: white;
+    padding: 50px;
+    display:flex;
+    justify-content: space-between;
+    background-color: #00B172;
+    background-image: url("../images/spotify-icon-white.png");
+    background-repeat: no-repeat;
+    background-size: 150px;
+    background-position: center;
+}
+.green-box .spotify-app img {
+    display: block;
```

Comment:
I don't see a change in layout if you remove the "display" property so  I think you don't need it here.